### PR TITLE
Refactor alert js

### PR DESCRIPTION
**Pull request by sttatusx**
Hi,
I refactored this file according to [this](https://sonarcloud.io/project/issues?open=AYjJwp0g5aesmyw3FN6R&id=jellyfin_jellyfin-web) sonar issue.

@thornbill

### CODE REVIEW

File: src/components/alert.js [hunk 1]
```diff
@@ -1,43 +1,36 @@
...
-
-function useNativeAlert() {
-    // webOS seems to block modals
-    // Tizen 2.x seems to block modals
-    return !browser.web0s
-            && !(browser.tizenVersion && browser.tizenVersion < 3)
-            && browser.tv
-            && window.alert;
-}
+import { appRouter } from "./router/appRouter";
```
Comment (thornbill) [inline src/components/alert.js:1]: Sorry for the delay! The eslint issues listed here need fixed. You should be able to run `npx eslint --fix` locally to fix them automatically. Thanks!

**Comment by sttatusx** [comment]
Fixed!

@thornbill

File: src/components/alert.js [hunk 2]
```diff
@@ -3,41 +3,34 @@ import browser from '../scripts/browser';
...
+    if (canUseNativeAlert) {
         alert((options.text || '').replaceAll('<br/>', '\n'));
+
         return Promise.resolve();
-    } else {
-        const items = [];
+    }
 
-        items.push({
+    const items = [
```
Comment (thornbill) [inline src/components/alert.js:25]: It doesn't look like this assignment is needed. We can just set `options.buttons` directly.

Comment (sttatusx) [reply src/components/alert.js]: You're right. Done!

**Review by thornbill — APPROVED** [review]

**Pull request merged**